trl/trainer/cpo_config.py

```diff
@@ -64,8 +64,12 @@ class CPOConfig(TrainingArguments):
 
     beta: float = 0.1
     label_smoothing: float = 0
-    loss_type: Literal["sigmoid", "hinge", "ipo", "kto_pair"] = "sigmoid"
+    loss_type: Literal["sigmoid", "hinge", "ipo", "kto_pair", "simpo"] = "sigmoid"
     disable_dropout: bool = True
+    simpo_gamma: float = 0.5
+    """
+    simpo_gamma: A target reward margin for the SimPO loss, used only when the "simpo" option is enabled.
+    """
```

Comment:
@fe1ixxu  i meant perhaps move this up into the docstrings?

Reply:
Sorry about the misunderstanding! I have moved it to the above docstrings!